srv/salt/_modules/osd.py

```diff
@@ -573,30 +573,31 @@ def wait(self):
         """
         i = 0
         last_pgs = 0
+        fmt_msg = msg = "osd.{} has {} PGs remaining"
```

Comment:
Is assigning `msg` intentional here?

Reply:
Yes, but only to avoid a NameError exception if for whatever reason we do not make at least one iteration through the `while` loop.

Perhaps it's better to explicitly set `msg` with a string such as "unknown error" ?

Reply:
I think that's a bit clearer, I'd vote for explicitly settinging `msg`